Competition2018/src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -57,11 +60,26 @@ public void setupElevatorCommands(
             LowerCommand lower,
             RiseCommand rise,
             CalibrateElevatorTicksPerInchCommand calibrateElevatorTicks,
-            CalibrateElevatorCommand calibrate) {
+            CalibrateElevatorViaStallCommand calibrate,
+            ElevatorMaintainerCommand maintainer,
+            SetElevatorTargetHeightCommand lowish,
+            SetElevatorTargetHeightCommand highish,
+            CalibrateElevatorHereCommand calibrateHere) {
         oi.operatorGamepad.getAnalogIfAvailable(oi.raiseElevator).whileActive(rise);
         oi.operatorGamepad.getAnalogIfAvailable(oi.lowerElevator).whileActive(lower);
         oi.operatorGamepad.getifAvailable(5).whileHeld(calibrateElevatorTicks);
+        oi.operatorGamepad.getifAvailable(6).whenPressed(maintainer);
         oi.operatorGamepad.getifAvailable(7).whenPressed(calibrate);
+        
+        lowish.setGoalHeight(20);
```

Comment:
Properties and/or express as a percentage

Reply:
Just hardcoded numbers to test things with a real robot - we still have work to do to identify key points, put properties for them, and then make the commands here.